packages/daemon/src/guest.js

```diff
@@ -49,16 +43,6 @@ export const makeGuestMaker = ({
       SELF: handleId,
       HOST: hostHandleId,
     });
-    const hostController =
-      /** @type {import('./types.js').EndoHostController} */
-      (await provideControllerAndResolveHandle(hostHandleId));
-    const hostPrivateFacet = await hostController.internal;
-    const { respond: deliverToHost } = hostPrivateFacet;
-    if (deliverToHost === undefined) {
-      throw new Error(
-        `panic: a host request function must exist for every host`,
-      );
-    }
```

Comment:
Dead code that was just laying around demanding to be refactored. It got the shovel instead.